node-js-web-server/server090.js

```diff
@@ -0,0 +1,15 @@
+const http = require('http');
+
+http.createServer(function (request, response) {
+   console.log('request url: ', request.url);
+   
+   var filePath = './public' + request.url;
+   if (filePath == './public/') {
```

Comment:
Just realized that this is using `==` rather than `===`. It's a minor detain, but we might consider changing this in the next draft.